File: HRO/CMITDW Taal der Wiskunde/tentamen/o2.java
public class o2
{
	public static int n = 600; // amount of terms
	public static int i = 2; // start from 3rd term
	
	public static void main(String[] args)
	{
		System.out.println("i\tn-2\tn-1\tn");
		func1(1, 3); // t1 = 1, t2 = 3
	}
	
	// (n * t(n-1)) / t(n-2)
	public static void func1(double a, double b)
	{
		i++;
		double c = (i*b) / a;
		System.out.println(i+"\t"+a+"\t"+b+"\t"+c);
		
		if (i < n)
			func1(b, c);
	}
}

File: HRO/CMITDW Taal der Wiskunde/tentamen/Lua/o2.lua
local n = 600 -- amount of terms
local i = 2 -- start from 3rd term

local function alg1(a, b)
	i = i + 1
	local c = (i*b) / a
	print(i, a, b, c)
	
	if i < n then
		alg1(b, c)
	end
end

print("i", "n-2", "n-1", "n")
alg1(1, 3)
